README.md

```diff
@@ -1,35 +1,42 @@
-Description
-===========
+# Nano Cookbook
+
+[![Build Status](https://travis-ci.org/chef-brigade/activemq.svg?branch=master)](https://travis-ci.org/chef-brigade/nano-cookbook) [![Cookbook Version](https://img.shields.io/cookbook/v/nano-cookbook.svg)](https://supermarket.chef.io/cookbooks/nano)
```

Comment:
I know that travis builds are not enabled for this repo, but I think this `activemq` here should be `nano-cookbook`.